src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
+
+
+const ChatEntry = ({sender, body, id, timeStamp, liked, updateChatData}) => {
+  const user ='Vladimir';
+  const [like, setLike] = useState(liked);
+  const userBubble = sender===user ? 'chat-entry local': 'chat-entry remote'
+
+  const handleLike = () => {
+      setLike(isLiked => {
+        const newState= !isLiked;
+        updateChatData(newState, id);
+        return newState;
+      })
+    };
```

Comment:
Taking the previous comment into account would actually eliminate the need for this handleLike function!